### ISSUE TITLE
How to ignore validation in disabled (dinamically) fieldset?

### ISSUE THREAD

**Issue by nelson6e65** [1000hz/bootstrap-validator#289]
I want to run validation on fields _only_ if they are enabled. I use a checkbox to enable/disable a fieldset.

Here is the runing code of a simple example I made: https://jsfiddle.net/nelson6e65/84yv9nkv/

**I noticed:**

- [x] :one:. I press  **Submit** without checking the `checkbox` before.<br />
It focuses on `Field 3` with error. :ok_hand: 
- [ ] :two:. I fill  `Field 3` without checking the `checkbox` before.<br /> 
Submit `button` is still disabled (css) (:raised_hand:) but form is sent if I click on it (:ok_hand:). 
- [x] :three:. I press **Submit** after click in the `checkbox`.<br />
Mark all fields with errors. :ok_hand:
- [x] :four:. I fill  `Field 3` and then I enable fields `1` and `2` by clicking in the `checkbox`.<br />
Fields `1` and `2` are marked with errors if I click on **Submit**. :ok_hand:
- [ ] :five:. After :four:, I disable again "Optional" `checkbox`.<br />
If I try to `Submit`, there are still with errors on those fields and submit `button` is disabled. I expect to be a valid, since those fields are disabled and should be ignored, right?. :raised_hand:

How can I solve :two: and :five:? :confused:

Or there is a way to skip validation in disabled fields?

**Comment by NestorLeone** [comment]
I have a similar problem....  I just "show" or "hide" the fieldset. 

Initially the fieldset is hidden, and validation returns OK even though all hidden fields are empty. They are required, so as soon as I display the filedset the validator tells me I need to fill them. 

Now when I hide the fieldset again, the validator still has the $("#dataForm").validator('validate').has('.has-error').length as 1.

**Comment by nelson6e65** [comment]
Yes... Maybe if I enable/disable each input in the fieldset manually with the checkbox `change` event... but that's not a very "glamorous" solution, I think. :disappointed:

**Comment by KevinLobo** [comment]
Same problem here!

**Comment by kunalmanchekar** [comment]
I am also facing same, please suggest solution.
